test/extended/testdata/test-imagesource-build.yaml

```diff
@@ -89,7 +89,7 @@ items:
           deploymentconfig: imagesourceapp
       spec:
         containers:
-        - image: imagesourceapp
+        - image: " "
```

Comment:
Didn't realize you made that change. It's fine for now, I will probably add one in the deployment suite at some point.